pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -593,7 +594,7 @@ def handle_command(
     if line[0] == "gfortran":
         from pyodide_build._f2c_fixes import replay_f2c
 
-        tmp = replay_f2c(line)
+        tmp = replay_f2c(build_args.f2c_path, line)
```

Comment:
You removed _f2c_fixes. how do we still call `_f2c_fixes.replay_f2c` here?

Reply:
I didn't remove it, I just deleted 450 out of 662 lines, the remaining 212 lines are still around.

More simplifications can probably be made in a followup but I removed all of the complicated stuff.

Reply:
Oops. GitHub collapsed the large diff, and I thought you removed it 🙄